connector/src/main/java/org/jboss/as/connector/subsystems/datasources/InstalledDriversListOperationHandler.java

```diff
@@ -97,18 +97,15 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
                         driverNode.get(DRIVER_MAJOR_VERSION.getName()).set(driver.getMajorVersion());
                         driverNode.get(DRIVER_MINOR_VERSION.getName()).set(driver.getMinorVersion());
                         driverNode.get(JDBC_COMPLIANT.getName()).set(driver.isJdbcCompliant());
-                        if (driver.getProfile() != null) {
-                            driverNode.get(PROFILE.getName()).set(driver.getProfile());
+                        if (profile != null) {
+                            driverNode.get(PROFILE.getName()).set(profile);
```

Comment:
Note that this is the same behavior was before, but I wonder if the profile attribute should be at least `UNDEFINED` if the profile is `null`. This would only be for standalone servers which currently don't have the attribute defined.